src/sonic-yang-models/tests/yang_model_tests/yangTest.json

```diff
@@ -1300,267 +1326,419 @@
+				"admin_status": "up",
+				"autoneg": "1",
+				"adv_speeds": "10000,25000",
+				"adv_interface_types": "CR,CR2",
+				"interface_type": "CR"
 			},
 			"Ethernet1": {
 				"alias": "Eth1/2",
 				"lanes": "66",
 				"description": "",
 				"speed": "11100",
-				"admin_status": "up"
+				"admin_status": "up",
+				"autoneg": "1",
+				"adv_speeds": "10000,25000",
+				"adv_interface_types": "CR,CR2",
+				"interface_type": "CR"
 			},
 			"Ethernet2": {
 				"alias": "Eth1/3",
 				"lanes": "67",
 				"description": "",
 				"speed": "11100",
-				"admin_status": "up"
+				"admin_status": "up",
+				"autoneg": "1",
+				"adv_speeds": "10000,25000",
+				"adv_interface_types": "CR,CR2",
+				"interface_type": "CR"
 			},
 			"Ethernet3": {
 				"alias": "Eth1/4",
 				"lanes": "68",
 				"description": "",
 				"speed": "11100",
-				"admin_status": "up"
+				"admin_status": "up",
+				"autoneg": "1",
+				"adv_speeds": "10000,25000",
+				"adv_interface_types": "CR,CR2",
+				"interface_type": "CR"
 			},
 			"Ethernet4": {
 				"alias": "Eth2/1",
 				"lanes": "69",
 				"description": "",
 				"speed": "11100",
-				"admin_status": "up"
+				"admin_status": "up",
+				"autoneg": "1",
+				"adv_speeds": "10000,25000",
+				"adv_interface_types": "CR,CR2",
+				"interface_type": "CR"
 			},
 			"Ethernet5": {
 				"alias": "Eth2/2",
 				"lanes": "70",
 				"description": "",
 				"speed": "11100",
-				"admin_status": "up"
+				"admin_status": "up",
+				"autoneg": "1",
+				"adv_speeds": "10000,25000",
+				"adv_interface_types": "CR,CR2",
+				"interface_type": "CR"
 			},
 			"Ethernet6": {
 				"alias": "Eth2/3",
 				"lanes": "71",
 				"description": "",
 				"speed": "11100",
-				"admin_status": "up"
+				"admin_status": "up",
+				"autoneg": "1",
+				"adv_speeds": "10000,25000",
+				"adv_interface_types": "CR,CR2",
+				"interface_type": "CR"
 			},
 			"Ethernet7": {
 				"alias": "Eth2/4",
 				"lanes": "72",
 				"description": "",
 				"speed": "11100",
-				"admin_status": "up"
+				"admin_status": "up",
+				"autoneg": "1",
+				"adv_speeds": "10000,25000",
+				"adv_interface_types": "CR,CR2",
+				"interface_type": "CR"
 			},
 			"Ethernet8": {
 				"alias": "Eth3/1",
 				"lanes": "73",
 				"description": "",
 				"speed": "11100",
-				"admin_status": "up"
+				"admin_status": "up",
+				"autoneg": "1",
+				"adv_speeds": "10000,25000",
+				"adv_interface_types": "CR,CR2",
+				"interface_type": "CR"
 			},
 			"Ethernet9": {
 				"alias": "Eth3/2",
 				"lanes": "74",
 				"description": "",
 				"speed": "11100",
-				"admin_status": "up"
+				"admin_status": "up",
+				"autoneg": "1",
+				"adv_speeds": "10000,25000",
+				"adv_interface_types": "CR,CR2",
+				"interface_type": "CR"
 			},
 			"Ethernet10": {
 				"alias": "Eth3/3",
 				"lanes": "75",
 				"description": "",
 				"speed": "11100",
-				"admin_status": "up"
+				"admin_status": "up",
+				"autoneg": "1",
+				"adv_speeds": "10000,25000",
```

Comment:
r they mandatory fields, we can remove those if not needed. We will try to keep the file small.

Reply:
I will remove them